### DISCUSSION TITLE
Using Polyvinyl Butyral (PVB) in FDS fire modelling

### DISCUSSION THREAD

**Discussion by jkjena1955** [firemodels/fds#13195] [Q&A]
Polyvinyl Butyral (PVB) is not a predefined species in FDS. As per user guide, th undefined species are replaced by nitrogen. I want to use PVB for the fire simulation of a bus. How to manage it?

**Comment by drjfloyd** [comment]
Per the Guide, thermal properties are set using the Lennard-Jones values for nitrogen and the assumption of gamma=1.4. You have the ability to define the actual properties via the various SPEC inputs discussed in the Guide.

**Comment by jkjena1955** [comment]
Thank you so much Dr Loyd!

Based on your advice, I took the input from FDS user manual for PVC and created the MATL and SURF. When i run, i get error on REAC line (exact copy from FDS User manual). Can you please advise me on the error and its resolution.
Defining PVC material 

&SPEC ID = 'PVC', FORMULA = 'C2H3Cl' /
&SPEC ID = 'OXYGEN', LUMPED_COMPONENT_ONLY = T /
&SPEC ID = 'NITROGEN', LUMPED_COMPONENT_ONLY = T /
&SPEC ID = 'HYDROGEN CHLORIDE', LUMPED_COMPONENT_ONLY = T /
&SPEC ID = 'WATER VAPOR', LUMPED_COMPONENT_ONLY = T /
&SPEC ID = 'CARBON MONOXIDE', LUMPED_COMPONENT_ONLY = T /
&SPEC ID = 'CARBON DIOXIDE', LUMPED_COMPONENT_ONLY = T /
&SPEC ID = 'SOOT', LUMPED_COMPONENT_ONLY = T /

&SPEC ID='AIR', BACKGROUND=T
SPEC_ID(1)='OXYGEN', VOLUME_FRACTION(1)=1.53,
SPEC_ID(2)='NITROGEN', VOLUME_FRACTION(2)=5.76 /

&SPEC ID='PRODUCTS',
SPEC_ID(1)='HYDROGEN CHLORIDE', VOLUME_FRACTION(1)=1.0,
SPEC_ID(2)='WATER VAPOR', VOLUME_FRACTION(2)=1.0,
SPEC_ID(3)='CARBON MONOXIDE', VOLUME_FRACTION(3)=0.14,
SPEC_ID(4)='CARBON DIOXIDE', VOLUME_FRACTION(4)=0.96,
SPEC_ID(5)='SOOT', VOLUME_FRACTION(5)=0.90,
SPEC_ID(6)='NITROGEN', VOLUME_FRACTION(6)=5.76 /

&INIT MASS_FRACTION(1)=0.229, SPEC_ID(1)='PVC' /

&REAC FUEL='PVC', HEAT_OF_COMBUSTION=16400., SPEC_ID_NU='PVC','AIR','PRODUCTS',
NU=-1,-1,1, FIXED_MIX_TIME=0.1 /

&MATL ID = 'PVC_MAT'
      SPECIFIC_HEAT = 0.9
      CONDUCTIVITY = 0.0002
      DENSITY = 1330.0
      NU_SPEC = 1.
      SPEC_ID = 'PVC'
      REFERENCE_TEMPERATURE = 175.0
      HEAT_OF_REACTION = 1795.
      HEAT_OF_COMBUSTION=16400./

&SURF ID ='PVC_Board'
      FYI            = 'Properties completely fabricated'
      BACKING        = 'VOID'
      COLOR          = 'BLUE'
      BURN_AWAY      = .TRUE.
      MATL_ID        = 'PVC_MAT'
      THICKNESS      = 0.005/

**Comment by drjfloyd** [comment]
FIXED_MIXED_TIME was moved to &COMB and we failed to edit that example to remove it. In general this isn't a parameter you would use.

**Reply by drjfloyd** [reply]
It looks like this oversight was corrected about a month ago and will be correct for the next release.
